docs/contributorfile.rst

```diff
@@ -132,75 +156,145 @@ Issues labels
 
 .. _g1i:
 
 Good First Issues
 ~~~~~~~~~~~~~~~~~
-Good First Issues are issues that are either very simple, or that help knowing the programs or the language better. We use it to help contributors with less experience to learn and familiarise with Git, GitHub, Python3, and physiology.
+Good First Issues are issues that are either very simple, or that help the contributor get to know the programs or the languages better. We use it to help contributors with less experience to learn and familiarise with Git, GitHub, Python3, and physiology.
 We invite more expert contributors to avoid those issues, leave them to beginners and possibly help them out in the resolution of the issue. However, if the issue is left unassigned or unattended for long, and it’s considered important or urgent, anyone can tackle it.
 
 .. _workflow:
 
 Contribution workflow
 ---------------------
 There are many descriptions of a good contribution workflow out there. For instance, we suggest to have a look at `tedana's workflow <https://github.com/ME-ICA/tedana/blob/master/CONTRIBUTING.md#making-a-change>`_.
-At physiopy, we follow a very similar workflow. The only two differences are:
+At ``physiopy``, we follow a very similar workflow. The only three differences are:
 
--  We ask you to test the code locally before merging it, and then, if possible, write some automatic tests for the code to be run in our Continuous Integration! Check the testing section below to know more.
--  We suggest opening a draft PR as soon as you can - so it’s easier for us to help you!
+- If you see an open issue that you would like to work on, check if it is assigned. If it is, ask the assignee if they need help or want to be substituted before starting to work on it.
+- We ask you to test the code locally before merging it, and then, if possible, write some automatic tests for the code to be run in our Continuous Integration! Check the testing section below to know more.
+- We suggest opening a draft PR as soon as you can - so it’s easier for us to help you!
 
 .. _pr:
 
 Pull Requests
 -------------
-To improve understanding pull requests "at a glance", we use the same labels used for issues. Multiple labels can be assigned - just think which ones suit your PR the most! 
-When opening a pull request, assign it to at least one label.
+To improve understanding pull requests "at a glance" and use the power of ``auto``, we use the labels listed above. Multiple labels can be assigned to a PR - in fact, all those that you think are relevant.
+We strongly advise to keep the changes you're introducing with your PR limited to your original goal. Little corrections here and there in the code that you're already modifying are a great help, but they risk complicating the nature of the PR and the reviewing process. It is much better to open another PR with the objective of doing such corrections!
```

Comment:
I don't agree with this. Why would we want to merge some code that we know needs corrections? We should not merge code that contains errors.

Another thing is that from that PR we think of another PR with a different goal. Then it makes sense to work on a new PR after merging.

Reply:
Let's see if the new rewording helps clarifying this point (commit to follow).